scripts/o3de/o3de/packaging.py

```diff
@@ -0,0 +1,318 @@
+#
```

Comment:
Feedback from the discussion: Let's make sure to include a task to write scoped automated tests (ideally unit tests) for anything we write here

Reply:
https://github.com/o3de/o3de/tree/development/scripts/o3de/tests